static/js/search_suggestion.js

```diff
@@ -40,6 +40,31 @@ function match_criteria(operators, criteria) {
     });
 }
 
+function compare_by_huddle(huddle) {
+    huddle = _.map(huddle.slice(0, -1), function (person) {
+        person = people.get_by_email(person);
+        if (person) {
+            return person.user_id;
+        }
+    });
+
+    var huddles = activity.get_huddles().slice(0, 10);
```

Comment:
Do we want to make a dictionary here for O(1) lookups?  It will make the rest of the code more concise, too.